pkg/pagerduty/types.go

```diff
@@ -0,0 +1,6 @@
+package pagerduty
+
+type Alert struct {
+	ID   string
+	Body map[string]interface{}
```

Comment:
this field is going to change and become more written out in the future?

Reply:
don't think so, but this is no longer something that is a single field

if you want I can chuch it to pagerduty.go